Formula/percona-xtrabackup.rb

```diff
@@ -0,0 +1,77 @@
+class PerconaXtrabackup < Formula
+  desc "Open source hot backup tool for InnoDB and XtraDB databases"
+  homepage "https://www.percona.com/software/mysql-database/percona-xtrabackup"
+  url "https://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.2/source/tarball/percona-xtrabackup-2.4.2.tar.gz"
+  sha256 "faeac6f1db4a1270e5263e48c8a94cc5c81c772fdea36879d1be18dcbcd1926e"
+
+  option "without-docs", "Build without man pages (which requires python-sphinx)"
+  option "without-dbd-mysql", "Build without bundled Perl DBD::mysql module, to use the database of your choice."
+
+  depends_on "cmake" => :build
+  depends_on "libev" => :build
+  depends_on "sphinx-doc" => :build if build.with? "docs"
+  depends_on :mysql => :build if build.with? "dbd-mysql"
```

Comment:
Picking up on Mike's suggested option wording though. `if build.with?` style dependencies are preferably avoided, and you're not losing anything with the simpler wording. I'll fix that in post so it doesn't need to go back through CI.